packages/react/src/components/FileUploader/stories/drop-container.js

```diff
@@ -5,18 +5,28 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, { useState, useCallback, useEffect } from 'react';
+import React, { useState, useCallback, useEffect, useRef } from 'react';
 import classnames from 'classnames';
 import FileUploaderItem from '../FileUploaderItem';
 import FileUploaderDropContainer from '../FileUploaderDropContainer';
 import FormItem from '../../FormItem';
-import uid from '../../../tools/uniqueId';
```

Comment:
Was there an issue with using the `uid` internal function like this? Just wondering why the internals were copied in here

Reply:
I'm not sure. I only moved this file over from the next folder out into the root. Looks like this is the implementation done during the v11 update.